### PR TITLE
Add base scope

### PR DESCRIPTION
**Pull request by darthjee**
This PR makes possible to add base scope for percentage checking

### CODE REVIEW

File: spec/lib/bidu/house/report/error_spec.rb [hunk 1]
```diff
@@ -153,16 +217,68 @@
...
 
-      it 'fetches from each scope in order' do
-        expect(subject.scoped.count).to eq(Document.with_error.type_b.count)
-        expect(subject.scoped.count).to eq(2 * Document.with_error.type_a.count)
+      context 'as symbol' do
+        let(:scope) { :'with_error.type_b' }
+
+        it 'fetches from each scope in order' do
+          expect(subject.scoped.count).to eq(Document.with_error.type_b.count)
+          expect(subject.scoped.count).to eq(2 * Document.with_error.type_a.count)
```
Comment (tadeusrox) [inline spec/lib/bidu/house/report/error_spec.rb:229]: why 2\* ?

Comment (darthjee) [inline spec/lib/bidu/house/report/error_spec.rb:229]: let(:types) { [:a, :b, :b] }

I forced 2 times b for each a

Comment (tadeusrox) [inline spec/lib/bidu/house/report/error_spec.rb:229]: AHHHH, thanks :)

**Comment by tadeusrox** [comment]
:shipit:

**Pull request merged**